tests/include/test/drivers/cipher.h

```diff
@@ -25,9 +25,10 @@ typedef struct {
     psa_status_t forced_status;
     /* Count the amount of times one of the cipher driver functions is called. */
     unsigned long hits;
+    unsigned long cipher_encrypt_hits;
```

Comment:
Yes, this is something I thought as well, but it would require to properly fix all the driver accesses' counts/checks in `test_suite_psa_crypto_driver_wrappers`. Should this be a good next task for the cleanup EPIC?